.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "souffle"]
```

Comment:
Why is this file removed? I need the submodule to easily update souffle headers every release.

Reply:
ok, reverting